src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -393,22 +393,15 @@ function clearElement(element: Element) {
 }
 
 function clearBetween(start: Node, end: Node): void {
-  const logicalParent = getLogicalParent(start as unknown as LogicalElement);
-  if (!logicalParent) {
-    throw new Error("Can't clear between nodes. The start node does not have a logical parent.");
-  }
-  const children = getLogicalChildrenArray(logicalParent);
-  const removeStart = children.indexOf(start as unknown as LogicalElement) + 1;
-  const endIndex = children.indexOf(end as unknown as LogicalElement);
-
-  // We remove the end component comment from the DOM as we don't need it after this point.
-  for (let i = removeStart; i <= endIndex; i++) {
-    removeLogicalChild(logicalParent, removeStart);
-  }
```

Comment:
Related to a change in `LogicalElements.ts`

Reply:
I think the existing logic will be required. Unless we use `removeLogicalChild`, the parent's list of logical children will no longer be in sync with the DOM.